daos/rinkeby/omentesting.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Omen Testing",
+  "Avatar": "0xd94c5810a43ae1560569592e041bf839e490cd1b",
+  "DAOToken": "0xc89dcfea2aef3a26a55e67a40593715bdea6ac4c",
+  "Reputation": "0xcae8cb5d5d9ceefd2a5f3e56fb33e6104e552bd8",
+  "Controller": "0xc2103ff0ae40dcbf4a61009998d96e2dd1b5cb75",
+  "Schemes": [
+    {
+      "name": "GenericScheme",
+      "alias": "GenericSchemeOmenDisputeResolution",
+      "address": "0x67883470ae73347a6742376d3db8bf13bc9358a8"
+    },
+    {
+      "name": "ExchangeTokenRegistry",
```

Comment:
this also a genericscheme ?

Reply:
@orenyodfat yes all the added schemes are Generics Schemes